topic04-git-introduction/book-git-tutorial/03.Rebasing-Without-Conflicts.md

```diff
@@ -0,0 +1,22 @@
+# *__Git rebase__*
+
+Push the changes by typing “git push <remote><branch>”, If the branch isn’t out of date, then the changes will push straight up to your github repo, If it is out of date however, you will have to rebase.
```

Comment:
I would just add a line to explain what out of date means.

Also I would add what an example 'git push' command would look like, ie `git pull origin master` for example